vuepress/docs/next/tutorials/devops/extend-plugin-descriptor.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebarDepth: 2
+---
+
+# Leverage Plugin Environment Variables 
+
+This tutorial describes how to use environment variables and secrets in the plugin descriptor  of an Entando Component Generator built project to customize microservices behavior. You can use the same docker image to run during development as in production by using the environment variables as a configuration mechanism for your pods or for passing sensitive information. A new validation step is embedded in the Entando process to manage possible incompatible properties.
```

Comment:
We can drop "of an Entando Component Generator-build project". Environment variables or secrets can be setup for any kind of plugin, generated or not.